packages/host/tests/integration/search-index-test.ts

```diff
@@ -1165,7 +1168,7 @@ module('Integration | search-index', function (hooks) {
     }
   });
 
-  test('search doc only includes used fields', async function (assert) {
+  test(`search doc includes 'contains' and 'containsMany' fields`, async function (assert) {
```

Comment:
what about a test for the search doc containing a used linksTo and linksToMany field? If there is a linksTo and linksToMay field that is in the template we should also be including that in the search doc

Reply:
updated